### ISSUE TITLE
DevContainer Support Is Missing For All Services

### ISSUE THREAD

**Issue by rizwaanbutt** [dotnet/eShop#120]
VSCode DevContainers is a very useful tool to setup and run the application very quickly. EShop does not support dev containers. Can we please add this support to the project?

**Comment by captainsafia** [comment]
@rizwaanbutt Thanks for opening this issue!

One thing to note WRT to devcontainer support is that the Aspire debugging experience is currently limited to VS. It's still possible to run the app host containing the application from the CLI in a dev container, but you won't get as full fidelity an experience with VS Code + dev containers.

Also, I'm not sure if it possesses an issue that Aspire takes a dependency on Docker. It is possible to run Docker inside a Docker container but not sure if Dev Containers make this difficult to experience.

All in all, I think we'll want to make sure the developer experience for Dev Containers actually works before incorporating it.

In the meantime, did you run into any issues while trying to setup and run this app?

**Comment by jamesmontemagno** [comment]
This should work somewhat today, you can run docker inside of the devcontainer... 

@SrushtiPasari94  can you add in devcontainer support and documentation if ncessary

**Comment by captainsafia** [comment]
> This should work somewhat today, you can run docker inside of the devcontainer...

Yep, the primary downside is that you won't be able to debug your Aspire app from VS Code since that isn't supported yet. But just running the application is possible.

@SrushtiPasari94 Let me know if you need any help getting started with this work.

**Comment by buzzaII** [comment]
Hey all.

I have successfully setup this to the point it will run the web app on devcontainers.

Is this something you still wanted to enable as this issue is >1 years old
